.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+**What I'm trying to do**
+
+[Clearly describe what you're trying to do.]
```

Comment:
This looks like a Bug report template. We might need a [Feature Request](https://help.github.com/en/articles/about-issue-and-pull-request-templates#issue-templates) template as well.

Reply:
Note: we don't have the permissions required to use the template builder, so we're stuck with the old way of adding templates

Reply:
@rolsonquadras is there any specific change you're requesting for this file?

Reply:
As you mentioned about the [permission issue](https://github.com/hyperledger/aries-framework-go/pull/75#discussion_r311748819), not sure we can do much. I'm ok with [patch](https://github.com/hyperledger/aries-framework-go/compare/f4423945b781038c7655a19c0dbe820aee0c6944..151000517d2512bebf2d96f64b648a8e9a36f796#diff-1a1c3dd142f76a5fad803a0c52839881R2) mentioning about feature request.

Reply:
We're calling feature requests "proposals". Is there anything else you'd like to be added?